index.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+require_once 'functions.php';
+
+$servername = "localhost";
+$username = "root";
+$password = "password";
+$db = "vinyl-collection";
+
+$db = retrieveDb();
+
```

Comment:
lines 11 - 19 are retrieving your data, these could be functionalised and put in your functions.php file